INSTALL.md

```diff
@@ -69,6 +69,24 @@ at [localhost:1081](http://localhost:1081).
 - Keep an eye on console in case of compiling errors.
 - [Read more](https://github.com/Trustroots/trustroots/wiki/Development)
 
+## Enable collecting statistics
+
+* 1. [Install InfluxDB](https://docs.influxdata.com/influxdb/latest/introduction/installation/) 0.9+ and run it (type `influxd`) at default port (8086)
```

Comment:
currently when no influxdb set, an error gets logged to the console each time the service tries to write a point. should it be removed?

i still feel a bit insecure about leaving the influx services running on their own (it gets decoupled from the main callback chain in the messages.server.controller (mainly for performance - faster response).

Any suggestions how to keep track of the possible errors, influxdb down etc? Somehow mailing admins would be one of the options.

These message statistics can be rebuilt from history, but at some point there will be other stats where it will be impossible. So to get any relevant output it should be up as much as possible.